src/medCore/gui/toolboxes/medRegistrationSelectorToolBox.cpp

```diff
@@ -207,14 +178,13 @@ void medRegistrationSelectorToolBox::changeCurrentToolBox(int index)
     }
 }
 
-
 //! Clears the toolbox.
 void medRegistrationSelectorToolBox::clear(void)
```

Comment:
I am indeed on VS, but i was looking for objects in the core that would call this method, which is a slot btw.